###MY_VER_RANGE [4.1.3_alpha,4.1.14_alpha20050808)

# MySQL Bugs: #11083: myisam.test fail w/ --without-geometry

--- mysql.orig/mysql-test/t/myisam.test	2005-05-26 16:55:48.000000000 +0200
+++ mysql.fix/mysql-test/t/myisam.test	2005-06-03 21:26:28.000000000 +0200
@@ -479,7 +479,7 @@
 #
 # Test RTREE index
 #
---error 1235
+--error 1235,1289
 CREATE TABLE t1 (`a` int(11) NOT NULL default '0', `b` int(11) NOT NULL default '0', UNIQUE KEY `a` USING RTREE (`a`,`b`)) ENGINE=MyISAM;
 # INSERT INTO t1 VALUES (1,1),(1,1);
 # DELETE FROM rt WHERE a<1;
--- mysql.orig/mysql-test/r/myisam.result	2005-05-26 16:55:48.000000000 +0200
+++ mysql.fix/mysql-test/r/myisam.result	2005-06-03 21:27:20.000000000 +0200
@@ -506,7 +506,7 @@
 1	SIMPLE	t2	index	NULL	PRIMARY	4	NULL	2	Using index; Distinct
 drop table t1,t2;
 CREATE TABLE t1 (`a` int(11) NOT NULL default '0', `b` int(11) NOT NULL default '0', UNIQUE KEY `a` USING RTREE (`a`,`b`)) ENGINE=MyISAM;
-ERROR 42000: This version of MySQL doesn't yet support 'RTREE INDEX'
+Got one of the listed errors
 create table t1 (a int, b varchar(200), c text not null) checksum=1;
 create table t2 (a int, b varchar(200), c text not null) checksum=0;
 insert t1 values (1, "aaa", "bbb"), (NULL, "", "ccccc"), (0, NULL, "");
